### ISSUE TITLE
[BUG] [SP-API] [Issue] MD9001: This developer is currently blocked for new mws authorizations

### ISSUE THREAD

**Issue by chris-skuvault** [amzn/selling-partner-api-docs#2916]
## I'm submitting a...

<pre><code>[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

We have a seller central account that we have disabled our listed application in and then went through the oauth flow to reauthorize and when its on the grant screen step and you click allow the app to be installed instead of redirecting back to the application we receive an error screen with the following message: 

`MD9001: This developer is currently blocked for new mws authorizations`

We cannot find any information about this error or code or what it might indicate. Currently our application is in the process of being relisted changing the API Type dropdown from being blank to SP-API so potentially until our app is relisted then Amazon is seeing it as a hybrid app still and in theory this might cause this scenario? 

The process of getting a listing to be republished is taking time and wanted to reach out to see if anyone knew any details about this error message and under what scenario might cause it to happen. We want to understand that what we are trying to do will resolve the issue we are running into.

**Comment by bradykieffer** [comment]
I'm also getting reports on this error and we haven't touched anything related to SP-API auth in months

**Comment by nrogers805** [comment]
> I'm also getting reports on this error and we haven't touched anything related to SP-API auth in months

Is your App a Hybrid ? Just trying to narrow the issue down.

**Comment by bradykieffer** [comment]
@nrogers805 we are hybrid yes

**Comment by nrogers805** [comment]
> @nrogers805 we are hybrid yes

Thanks for confirming

Have you updated it to full SP-API as suggested in the latest update from Amazon? 

We have but now the error seems to be random. Some are able to authorise and some still get the erros

Amazon are fuckwits

**Comment by repriceit** [comment]
We're having the same issue.  We updated to full SP-API, and re-published our app as Amazon requested in their email, but so far all new signups today are getting the message, "This developer is currently blocked for new mws authorizations".  Very frustrating.  Not sure how long the changes take to be in effect.  We have an open ticket, but no response yet.

**Comment by nrogers805** [comment]
> 



> We're having the same issue. We updated to full SP-API, and re-published our app as Amazon requested in their email, but so far all new signups today are getting the message, "This developer is currently blocked for new mws authorizations". Very frustrating. Not sure how long the changes take to be in effect. We have an open ticket, but no response yet.

yes, very frustrating - and as usual, nobody from Amazon are supporting these threads - dreadful service as usual

**Comment by sungolivia** [comment]
hey everyone, thanks @nrogers805 thanks for the kind nudge, too bad I am here! 😝 

I would check few things:

If your app is pure SP-API,
1. if you have submitted your app for publishing to pure SP-API app, it takes awhile so make sure your status is changed to `published` not `pending`. If it's still `pending`, you will see this error
2. if you have unpublished changes, most likely the app will not be published so go check the app status once again!
3. Please don't use Auth API as it's only for to be used in hybrid app status

Any other update I will provide as I get updated as well.

**Comment by nrogers805** [comment]
> hey everyone, thanks @nrogers805 thanks for the kind nudge, too bad I am here! 😝
> 
> I would check few things:
> 
> If your app is pure SP-API,
> 
> 1. if you have submitted your app for publishing to pure SP-API app, it takes awhile so make sure your status is changed to `published` not `pending`. If it's still `pending`, you will see this error
> 2. if you have unpublished changes, most likely the app will not be published so go check the app status once again!
> 3. Please don't use Auth API as it's only for to be used in hybrid app status
> 
> Any other update I will provide as I get updated as well.

As I already said, this was working fine up until today.

the staus is :Published
Current edit is in draft

its been like this for months , so should be ok today

**Comment by nrogers805** [comment]
I am not using poi

> > hey everyone, thanks @nrogers805 thanks for the kind nudge, too bad I am here! 😝
> > I would check few things:
> > If your app is pure SP-API,
> > 
> > 1. if you have submitted your app for publishing to pure SP-API app, it takes awhile so make sure your status is changed to `published` not `pending`. If it's still `pending`, you will see this error
> > 2. if you have unpublished changes, most likely the app will not be published so go check the app status once again!
> > 3. Please don't use Auth API as it's only for to be used in hybrid app status
> > 
> > Any other update I will provide as I get updated as well.
> 
> As I already said, this was working fine up until today.
> 
> the staus is :Published Current edit is in draft
> 
> its been like this for months , so should be ok today

I am using the SP-API workflow and nothing to do with MWS

**Comment by nrogers805** [comment]
> 

Also, some sellers are able to go through the process successfully. I can also run through the process successfully on my  own account. But so many sellers are getting this message - its obviously an Amazon issue but getting no response from support at all

**Comment by sungolivia** [comment]
oh make sure to cut our support team tickets as they won't be able to help without your app IDs!
https://developer.amazonservices.com/support

**Comment by nrogers805** [comment]
> oh make sure to cut our support team tickets as they won't be able to help without your app IDs! https://developer.amazonservices.com/support
> 
> currently our team is aware and looking into it so definitely let us know via the support ticket if you are affected!

I have already raised a ticket , 15 hours ago, and no response, which is appalling, especially if its a known issue. I provided my app id.

If its a known issue then we should have been broadcasted by email, again dreadful service for such an issue

**Comment by nrogers805** [comment]
This seems to have been fixed by Amazon, as new sellers are now able to authorise. 

But still not had any reply from SP-AP support!

**Comment by chris-skuvault** [comment]
> This seems to have been fixed by Amazon, as new sellers are now able to authorise.
> 
> But still not had any reply from SP-AP support!

Yep seems to be fixed on our end as well now. Very frustrating we put a support ticket in about this last week and really heard nothing back other than they are investigating internally for days. I guess they resolved it now.

One thing to note is that we are still pending approval to relist our app after switching it from "" to SP-API which I assumed was what was needed to resolve this issue but it seems like since it hasn't been relisted then it was just an issue amazon had on their end.

Going to close this ticket as the issue seems to be resolved.

**Comment by jamiethompson** [comment]
We've started experiencing this error this week. On Monday Amazon emailed all of our sellers to tell them that our MWS access was going to be revoked on 4/4/23. But that email listed several developer IDs, none of which were ours. We also no longer use any MWS functionality. We have been SP-API only since late last year.

We're trying to understand what is happening. Our app was set to Hybrid. I've changed that to SP-API only today. We never had any communication from Amazon that this needed to be changed. Changing this setting however doesn't seem to have had any effect.
